### ISSUE TITLE
Entity Not Found

### ISSUE THREAD

**Issue by matrixjnr** [vaadin-learning-center/crm-tutorial#44]
When trying to move around the project, I had some issues with Entity Not Found Exception from Company Entity.
Probably I would suggest adding (fetch = FetchType.EAGER,mappedBy = "company",cascade = CascadeType.ALL) for the relationship.

I had to revert to the normal Integer id instead of uuid. Seems like the generated UUID was not being mapped correctly between the contact and the company table.

**Comment by tarekoraby** [comment]
Did you fix the issue? Could you clarify what you mean by "When trying to move around the project"?

**Comment by matrixjnr** [comment]
Hello,

I was trying to create a sample Vaadin project. So while running the
project, at first it was okay. But later on, the generated data when the
project was being started would not match i.e the relationship between the
Contacts and Company did not seem to work. Some contacts had references to
company_id that did not exist. I switched from UUID to normal integer id
and the project was okay for the next 10+ times I tried running it. But for
a normal database created without generated data, the project is okay. That
is why I closed the issue.
